Documentation/Makefile

```diff
@@ -45,7 +45,32 @@ copy-api:
 	@$(ECHO_GEN)_api
 	$(QUIET)cp -r ../api _api
 
-epub latex html: builder-image copy-api
+HELM_VALUES = helm-values.rst
+
+HELM_DOCS_VERSION := v1.3.0
+HELM_DOCS_SHA := 12f065115654c9a1d6819b938487d327f1fdb5c32bdc2a5c74afef22ed68f5ca
+HELM_DOCS_IMAGE := docker.io/jnorwood/helm-docs:$(HELM_DOCS_VERSION)@sha256:$(HELM_DOCS_SHA)
+HELM_DOCS_ROOT_PATH := /src
+HELM_DOCS_CHARTS_DIR := $(HELM_DOCS_ROOT_PATH)/install/kubernetes
+HELM_DOCS_OUTPUT_DIR := $(HELM_DOCS_ROOT_PATH)/Documentation
+HELM_DOCS := $(DOCKER_CTR) -v $(PWD)/..:$(HELM_DOCS_ROOT_PATH) $(HELM_DOCS_IMAGE)
+
+M2R_VERSION := 0.2.1
+M2R_SHA := a3dc2c3fcdc31260dcb7cd42e39763281a6fdce4ca59b4c150fb3bfff5be9eb7
+M2R_IMAGE := docker.io/bmcustodio/m2r:$(M2R_VERSION)@sha256:$(M2R_SHA)
+M2R := $(DOCKER_CTR) $(M2R_IMAGE) m2r
+
+.PHONY: $(HELM_VALUES)
+$(HELM_VALUES): TMP_FILE := helm-values.tmp
+$(HELM_VALUES): AWK_FILE := helm-values.awk
+$(HELM_VALUES):
+	$(QUIET)touch $(TMP_FILE) $(AWK_FILE)
```

Comment:
The `touch` seems unnecessary. I think I had an error on the first time because you wouldn't generate the temporary file at the same path as you would try to read? Maybe something to do with the `PWD`/`CURDIR`. But it works for me without this `touch` even if the file does not exist beforehand, as long as the files are correctly generated before we call `M2R`.

Reply:
(Conversation marked as resolved... but the `touch` is still present?)